### PR TITLE
Add script to launch tfjs debugger local server

### PR DESCRIPTION
**Pull request by jinjingforever**
The script will pull the zip bundle from cloud storage, unzip it, and serve its files from memory. It also serves other files from the tfjs repo.

The script is supposed to be run from the root of the tfjs repo.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047)

### CODE REVIEW

**Review by mattsoulanille — APPROVED** [review]
LGTM with a tiny nit.

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@types/shelljs": "^0.8.7",
     "@types/webgl-ext": "0.0.30",
     "@webgpu/types": "0.1.21",
+    "adm-zip": "0.5.9",
```
[the remark this answers is not in the document]
Comment (jinjingforever) [reply package.json]: Good know! Done. Thank you!

**Review by pyu10055 — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @jinjingforever and @mattsoulanille)

___
*[`scripts/start_local_debugger_server.js` line 23 at r3](https://reviewable.io/reviews/tensorflow/tfjs/7047#-NGsAh3B6c3lmkuxwnCC:-NGsAh3B6c3lmkuxwnCD:b7b3ze9) ([raw file](https://github.com/tensorflow/tfjs/blob/3c79fabfcacf9cb78635de3306dcad4687e214ce/scripts/start_local_debugger_server.js#L23)):*
> ```JavaScript
> 
> const TFJS_DEBUGGER_BUNDLE_URL =
>     'https://storage.googleapis.com/tfweb/tfjs-debugger-bundle/tfjs-debugger_20221111-105432.zip';
> ```

does cloud storage allow symlink? can we have a something like
`tfjs-debugger-bundle/tfjs-debugger_latest.zip` that allows the script to use the latest build?
or should the arg parser be able to parse command line version?

**Review by pyu10055 — COMMENTED** [review]
LGTM with a nit as well.

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047#-:-NGsG4ks0YIS5594J69X:b-c9xz4l)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @jinjingforever and @mattsoulanille)

**Review by jinjingforever — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @mattsoulanille and @pyu10055)

___
*[`scripts/start_local_debugger_server.js` line 23 at r3](https://reviewable.io/reviews/tensorflow/tfjs/7047#-NGsAh3B6c3lmkuxwnCC:-NH6Ck-EEM5p-OXfphOp:be3ohya) ([raw file](https://github.com/tensorflow/tfjs/blob/3c79fabfcacf9cb78635de3306dcad4687e214ce/scripts/start_local_debugger_server.js#L23)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

does cloud storage allow symlink? can we have a something like
`tfjs-debugger-bundle/tfjs-debugger_latest.zip` that allows the script to use the latest build?
or should the arg parser be able to parse command line version?
</blockquote></details>

Looks like the symlink support in storage bucket is limited. I added a --version flag (with a default value) as you suggested. Thank you for the idea!

**Comment by jinjingforever** [comment]
Hi @pyu10055 , looks like I need the approval to merge this. Thanks!

**Review by pyu10055 — APPROVED** [review]
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @mattsoulanille)

**Review by pyu10055 — COMMENTED** [review]
approved, sorry for the delay.

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7047#-:-NI3M65i2wDB1_4ejRMs:b-ib2an0)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @mattsoulanille)

**Pull request merged**